src/components/NewsSection.js

```diff
@@ -1,22 +1,44 @@
 import styled from 'styled-components';
 import NewsArticleHeadLine from './NewsArticleHeadline';
 import NewsArticle from './NewsArticle';
+import news from '../utils/newsData';
 
 export default function NewsSection() {
   return (
     <NewsWrapper>
       <NewsTitle>News</NewsTitle>
-      <NewsArticleHeadLine />
-      <NewsArticle />
+      <NewsArticleList>
+        {news.map((articles) => {
+          return (
+            <li key={articles._id}>
+              <NewsArticleHeadLine
+                headline={articles.headline}
+                date={articles.date}
+              />
+              <NewsArticle text={articles.text} />
```

Comment:
You could write it like below :) 
```suggestion
        {news.map(({ _id, headline, date, text}) => {
          return (
            <li key={_id}>
              <NewsArticleHeadLine
                headline={headline}
                date={date}
              />
              <NewsArticle text={text} />
```